app/controllers/admin/tags_controller.rb

```diff
@@ -38,6 +44,7 @@ def edit
     def update
       @tag = Tag.find(params[:id])
       if @tag.update(tag_params)
+        ::Tags::AliasRetagWorker.perform_async(@tag.id) if tag_alias_updated?
```

Comment:
There may be a lot of tags, so an async worker seemed like the right choice. The worker is supposed to be idempotent, so it's possible to restart it if it gets interrupted for any reason.